docs/samples/edr-api-overview/edr-api-overview.md

```diff
@@ -0,0 +1,136 @@
+# Endpoint Data Reference API Overview
+
+## Introduction
+
+The **Endpoint Data Reference** serves the purpose of streamlining the interactions between consumer connector operators. Through its API, acts as a facade for two fundamental processes involved in Asset consumption: **contract negotiation** and **transfer process** initialization. Additionally, it enables the management of **EDR**s (Endpoint Data References).
+
+## Configuration
+
+The new API is seamlessly deployed under the well-known management API context, requiring no additional configuration. However, this is not the case for the EDR Cache storage. The EDR Cache provides two implementations: `InMemory` and `SQL`. The `edc-runtime-memory` bundle utilizes the `InMemory` implementation for the EDR Cache without any extra setup. On the other hand, the SQL Cache, found in the `edc-controlplane-postgresql-*` bundles, utilizes the SQL implementation and requires database access for schema migration and data management.
+
+| property                                                  | description                                          | required  | default value |
+|-----------------------------------------------------------|------------------------------------------------------|-----------|---------------|
+| edc.datasource.edr.name                                   | Defines the name associated with the EDR data source | false     | edr           |
+| edc.datasource.edr.url                                    | Defines the database address to access the EDR data  | true      |               |
+| edc.datasource.edr.user                                   | Defines EDR database username                        | true      |               |
+| edc.datasource.edr.password                               | Defines EDR database password                        | true      |               |
+| org.eclipse.tractusx.edc.postgresql.migration.edr.enabled | Enables EDR database migration	                      | true      |               |
+
+## Features
+
+### EDR Negotiation | Contract Negotiation and Transfer Process in a single request
+
+- This endpoint will perform the contract negotiation, transfer process and EDR storage respectively.
+
+> Please note that the `data destination` will always be `HttpProxy`, requiring a request against the provider's `data-plane` to fetch the asset data.
+
+| Path                            | Method | Query Params             |
+|---------------------------------|--------|--------------------------|
+| `<MANAGEMENT_URL>/edrs`         | POST   | none                     |
+
+#### Payload
+
+```json
```

Comment:
NIT: The formatting of the placeholders seems strange to me, maybe just use `"connectorAddress": "<PROVIDER_DSP_URL>"` instead?